pkg/installation/installation.go

```diff
@@ -81,25 +82,41 @@ func (i *Installation) newStep(msg string) step.Step {
 }
 
 // InstallKyma triggers the installation of a Kyma cluster.
-func (i *Installation) InstallKyma() (*Result, error) {
+func (i *Installation) InstallKyma(actionName string) (*Result, error) {
```

Comment:
A function called install Kyma that can either install or upgrade depending on a string parameter is confusing, not flexible and very error prone.

It would be better to have 2 separate functions and encapsulate common behaviour.